content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -113,7 +113,7 @@ directory should be rotated. In Kubernetes clusters brought up by
 the `kube-up.sh` script, those logs are configured to be rotated by
 the `logrotate` tool daily or once the size exceeds 100MB.
 
-[glog]: https://godoc.org/github.com/golang/glog
+[klog]: https://godoc.org/github.com/kubernetes/klog
```

Comment:
I think the correct link would be https://godoc.org/k8s.io/klog or https://github.com/kubernetes/klog.

Reply:
Thanks for your reply. I will update.